.env.example

```diff
@@ -4,12 +4,9 @@ APP_DEBUG=true
 APP_LOG=daily
 APP_LOG_LEVEL=debug
 APP_URL=http://rogue.test
-ROGUE_API_KEY=abc123
 
-# Feature Flags
+# Feature Flags:
 DS_ENABLE_BLINK=false
-DS_ENABLE_GLIDE=true
-DS_ENABLE_V3_QUANTITY_SUPPORT=false
```

Comment:
These feature flags no longer exist! 🎏

Reply:
I removed this over in #980 -- happy to close that PR out instead, although it seems like maybe we'd want to also remove `app.json` here too?

Reply:
Oh nice! I'll rebase to fix that conflict and knock these others out of `app.json`.